src/mmw/apps/modeling/views.py

```diff
@@ -494,3 +494,58 @@ def boundary_layer_detail(request, table_code, obj_id):
             return Response(geojson)
         else:
             return Response(status=status.HTTP_400_BAD_REQUEST)
+
+
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def plot_point_source(request):
+    try:
+        query = '''
+              SELECT ST_AsGeoJSON(geom) as geometry, city, state, npdes_id,
+                     mgd, kgn_yr, kgp_yr
+              FROM ms_pointsource_drb
+              UNION
+              SELECT ST_AsGeoJSON(geom) as geometry, city, state, npdes_id,
+                     mgd, kgn_yr, kgp_yr
+              FROM ms_pointsource
+              '''
+    except (KeyError, IndexError):
+        return Response(status=status.HTTP_404_NOT_FOUND)
+
+    point_source_results = {
+        'type': 'FeatureCollection',
+        'features': []
+    }
+
+    with connection.cursor() as cursor:
+        cursor.execute(query)
+        columns = [col[0] for col in cursor.description]
+
+        point_source_array = [
+            dict(zip(columns,
+                     [row[0],
+                      row[1],
+                      row[2],
+                      row[3],
+                      float(row[4]) if row[4] else None,
+                      float(row[5]) if row[5] else None,
+                      float(row[6]) if row[6] else None]))
```

Comment:
Since the columns aren't dynamic, consider hard-coding the column->value mapping. This might make it easier to see at a glance that everything lines up correctly.

Ex.

```py
    point_source_array = [
        {
            'geometry': row[0],
            'city': row[1],
            # ...
        }
        for row in cursor.fetchall()
    ]
```